src/lib/accounts.ts

```diff
@@ -19,13 +19,7 @@ export async function loadAccount(slug: string, token: string) {
     slug,
   }
 
-  const authorizationHeader = token && {
-    authorization: `Bearer ${token}`,
-  }
-
-  graphQLClient.setHeaders({
-    ...authorizationHeader,
-  })
+  setAuthorizationHeaderForClient(graphQLClient, token)
```

Comment:
`setHeaders` will wipe all the existing headers so I added a helper function that will add the single header to the existing ones.

Reply:
another approach would be to `const graphQLClient = getGraphClient(token)` vs importing the instance and then updating it later.